app/helpers/application_helper.rb

```diff
@@ -93,17 +93,22 @@ def icon_url(name)
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
     return url if Rails.env.development? && (url.blank? || url.exclude?("http"))
 
-    service_path = "https://res.cloudinary.com/practicaldev/image/fetch"
-
-    if url&.size&.positive?
-      if width
-        "#{service_path}/c_scale,fl_progressive,q_auto,w_#{width}/f_auto/#{url}"
-      else
-        "#{service_path}/c_scale,fl_progressive,q_auto/f_auto/#{url}"
-      end
+    if url&.blank?
+      url = "https://pbs.twimg.com/profile_images/481625927911092224/iAVNQXjn_normal.jpeg"
```

Comment:
Where does this image come from, and can we host a blank image from the repo rather than externally?